src/app/features/permits/components/organisms/PermitOverview/PermitOverview.tsx

```diff
@@ -1,9 +1,9 @@
 import React, { useMemo } from "react"
-
+import { Link } from "@datapunt/asc-ui"
```

Comment:
I think `Link` from `@reach/router` should be used here. Check other `Link`s in the codebase.

Reply:
Oh, never mind. Now see it's an external link.

Reply:
Why is that? I want the one from ASC (https://amsterdam.github.io/amsterdam-styled-components/?path=/docs/ui-typography-link--default-story), because of the styling and icons.